Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:10
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
```

Comment:
You should be able to use
`npm ci --production`

Installs packages from package-lock.json, does not cache and --production does not install devDependencies.

Reply:
@Revadike This needs a package-lock.json though.

Reply:
> @Revadike This needs a package-lock.json though.

Interesting. Perhaps make this PR edit .gitignore to remove `package-lock.json` in it.